pipeline_steps/common.groovy

```diff
@@ -30,13 +30,6 @@ def void create_workspace_venv(){
     # UG-613 change TMPDIR to directory with more space
     export TMPDIR="/var/lib/jenkins/tmp"
 
-    # UG-612 Install pip, setuptools, and wheel on the host
```

Comment:
Currently centos slaves are not allocated unless explicitly requested. The only time we request centos slaves is when we need to access to the Jenkins REST API (eg for connecting a new slave).